core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunction.java

```diff
@@ -85,6 +86,14 @@ public static void register(BuiltinFunctionRepository repository) {
     repository.register(tan());
   }
 
+  private static DefaultFunctionResolver doubleMathFunctionBase(FunctionName functionName, SerializableFunction<Double, Double> formula) {
```

Comment:
I think it would be nice to have a Javadoc comment for this function

Reply:
I have added a Javadoc comment